iam/api/QuickStartV2Test/QuickStartTest.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Iam.v1;
+using Google.Apis.Iam.v1.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+
+namespace GoogleCloudSamples
+{
+    public class QuickStartTest : IDisposable
+    {
+        private readonly string _projectId;
+        private ServiceAccount serviceAccount;
+        private IamService iamService;
+
+        public QuickStartTest()
+        {
+            // Check for _projectId and throw exception if empty
+            _projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+            if (_projectId == null)
+            {
+                throw new System.ArgumentNullException("GOOGLE_PROJECT_ID", "Environment variable not set");

```

Comment:
nit: remove the System. it's already on the using clauses.

Reply:
Done